easybuild/easyconfigs/a/Artemis/Artemis-16.0.18-Java-1.8.0_152.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'JAR'
+
+name = 'Artemis'
+version = '16.0.18'
+
+homepage = 'http://www.sanger.ac.uk/science/tools/artemis'
+description = """Artemis is a free genome browser and annotation tool that allows visualisation of sequence features,
+ next generation data and the results of analyses within the context of the sequence, and also its six-frame
+ translation."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/v16/v%(version)s']
+sources = ['%(namelower)s-v%(version)s.jar']
+
+java = 'Java'
+javaver = '1.8.0_152'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [
+    (java, javaver),
+    ('X11', '20160819', '-foss-2015b'),
```

Comment:
We usually do it the other way, by specifying `foss/2015b` via `toolchain`.

Even though nothing gets compiled when using the `JAR` easyblock, it's just cleaner that way (and more consistent with other easyconfigs).

In addition, please use the `%(javaver)s` template and move `versionsuffix` up right below `version`, like this:

```python
name = 'Artemis'
version = '16.0.18'
versionsuffix = '-Java-%(javaver)s'

...

dependencies = [
    ('X11', '20160819'),
    ('Java', '1.8.0_152'),
]
```